flexmeasures/data/services/utils.py

```diff
@@ -55,10 +55,16 @@ def hash_function_arguments(args, kwags):
     )  # concat two hashes
 
 
-def redis_cache(queue):
+def job_cache(queue):
     """
-    Decorator that checks if a function has already been called with the same arguments and
-    fetches the job using the job id, which is stored in Redis.
+    To avoid recomputing the same task multiple times, this decortator checks if the function was already been called with the
```

Comment:
decortator -> decorator

Reply:
ups, haha, thanks for spotting them, I'll try to make less :S

Reply:
Just waiting for my other PR to land!